styles.css

```diff
@@ -259,21 +250,429 @@
+  flex: none;
+  order: 2;
+  flex-grow: 0;
+  padding-top: 10px;
+  text-align: center;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-size: 15px;
+  line-height: 25px;
+}
+
+.ruby {
+  height: 40px;
+  width: 61px;
+  left: 236px;
+  top: 0;
+  list-style: none;
+  text-align: center;
+  background: #fff;
+  border: 1px solid #8993a4;
+  flex: none;
+  order: 3;
+  flex-grow: 0;
+  padding-top: 10px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 25px;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+  margin-top: -70px;
+}
+
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 386px;
+  width: 327px;
+  margin-top: 12px;
+  background: url('images/bckImg.png');
+  background-repeat: no-repeat;
+  background-size: 340px auto;
+}
+
+.card h4 {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  padding: 48px 0 0 16px;
+  margin-bottom: 12px;
+  font-weight: 400;
+}
+
+.card p {
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  width: 295px;
+  padding: 0 0 0 16px;
+  line-height: 24px;
+  color: #fff;
+  margin-top: 0;
+}
+
+.tags li a {
+  text-decoration: none;
+  color: #fff5e1;
+}
+
+.tags ul {
+  margin-top: -10px;
+  width: 236px;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1px 12px;
+}
 
-  left: 282.1px;
-  top: 590px;
-  position: absolute; */
+.tags li {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  font-family: 'Inter', sans-serif;
+  color: rgb(23, 167, 13);
+  padding: 10px 12px;
+  background: rgba(255, 255, 255, 0.24);
+}
+
+.btn-2 {
+  margin-top: 10px;
+  padding: 12px;
+  gap: 10px;
+  width: 100%;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 700;
+  color: white;
+  border: none;
+}
+
+.btn-2:hover {
+  color: #f55800;
+  background: #fff5e1;
+  border: 1px solid #f55800;
+}
+
+.AboutMe {
+  height: 563px;
+  width: 375px;
+  left: 0;
+  top: 4137px;
+  background: #1c1a19;
+}
+
+.AboutMe h1 {
+  font-family: 'Crete Round', sans-serif;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  padding: 10px 24px 24px 24px;
+  margin-bottom: 24px;
+}
+
+.AboutMe p {
+  color: #f4f5f7;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding: 24px 28px 24px 24px;
+  letter-spacing: 0;
+  text-align: left;
+  margin-top: -50px;
+}
+
+.AboutMe button {
+  background: #ff6b00;
+  padding: 12px;
+  color: #fff;
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  margin-left: 24px;
+  margin-top: -50px;
+  outline: #ff6b00;
+}
+
```

Comment:
- I think it will be great if you implemented the hover effect on your `about me` button, just as you did with the `see project` for each card. 🍷 

What do you think about this??